spec/unit/views/components/status_tag_spec.rb

```diff
@@ -20,7 +20,7 @@ def status_tag(*args)
 
     describe '#class_list' do
       subject { super().class_list }
-      it      { is_expected.to include('status_tag') }
+      it      { is_expected.to include('status-tag') }
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.